contract-examples/test/reward_manager.ts

```diff
@@ -119,28 +114,6 @@ describe("ExampleRewardManager", function () {
     expect(balance.gt(previousBalance)).to.be.true
   })
 
-  it("signer1 should be appointed as reward address", async function () {
-    let tx = await contract.setRewardAddress(signer1.address);
-    await tx.wait()
-    let rewardAddress = await contract.currentRewardAddress();
-    expect(rewardAddress).to.be.equal(signer1.address)
-  });
-
-  it("signer1 should be able to receive fees", async function () {
-    let previousBalance = await ethers.provider.getBalance(signer1.address)
-
-    // Send a transaction to mine a new block
-    const tx = await owner.sendTransaction({
-      to: signer2.address,
-      value: ethers.utils.parseEther("0.0001")
-    })
-    await tx.wait()
-
-    let balance = await ethers.provider.getBalance(signer1.address)
-    expect(balance.gt(previousBalance)).to.be.true
-  })
```

Comment:
why have we removed those? these are testing the actual logic in reward manager precompile.

Reply:
ah ok that was a redundant test. Some tests were put to clear possible confusions like "reward address should be same address as the wrapping contract" etc. so specifically put that one to show EOAs & contracts can both be reward address. but thats a reduntant test indeed.